src/styles/styles.js

```diff
@@ -3798,6 +3798,13 @@ const styles = (theme) => ({
         lineHeight: variables.lineHeightLarge,
     },
 
+    walletBalance: {
+        lineHeight: 'normal',
+        fontSize: 45,
```

Comment:
Can you check this against the other font sizes variables we have in our styles? Maybe we can reuse an existing variable for this instead of writing something new.

Reply:
@shawnborton I did verify but I didn't find any